mettle/src/tlv.c

```diff
@@ -574,18 +568,16 @@ int tlv_dispatcher_process_request(struct tlv_dispatcher *td, struct tlv_packet
 /*
  * This works around garbage on the socket by attempting to read past it.
  */
-bool tlv_have_sync_packet(struct buffer_queue *q, const char *method)
+bool tlv_found_first_packet(struct buffer_queue *q)
 {
 	bool found = false;
-	size_t method_len = strlen(method);
 
-	while (buffer_queue_len(q) >= 62 + method_len) {
+	while (buffer_queue_len(q) >= 571) {
```

Comment:
Seems a bit tenuous to hard code a value here. Do we have enough on this side of the codebase to know from a const or dynamic var what this should be?